src/components/ModiteList/index.tsx

```diff
@@ -17,36 +17,60 @@ import ModiteListProps from '../../models/ModiteListProps';
 import SkeletonList from '../SkeletonList';
 import ModiteListItem from '../ModiteListItem';
 import ModitesContext from '../../state/modites';
+import Project from '../../models/Project';
+import ModiteContext from '../../state/modite';
+import DetailsView from '../../components/DetailsView';
 
 // get locale once
 const locale: string = navigator.language;
 
 // reference to the worker that formats and filters modite data
 const worker: Worker = new Worker();
 
-// keep server response here for future reference
+// keep server response for Modites here for future reference
 let rawModites: Modite[];
+// keep server response for Projects here for future reference
+let rawProjects: Project[];
+// points the the active raw list data: rawModites or rawProjects
+let rawListSource: Modite[] | Project[];
 
 // get data from server
-async function getData(filter: string, date: Date): Promise<void> {
+async function getModiteData(filter: string, date: Date): Promise<void> {
   if (!rawModites || !rawModites.length) {
-    const moditesResp: { modites: Modite[] } = await fetch('https://modus.app/modites/all').then(res => res.json());
-    rawModites = moditesResp.modites;
+    const fetches = [
+      new Promise(async resolve => {
+        const moditesResp: Modite[] = await fetch('https://modus.app/modites/all').then(res => res.json());
+        rawModites = moditesResp;
+        resolve();
+      }),
+      new Promise(async resolve => {
+        const projectsResp: Project[] = await fetch('https://modus.app/projects/all').then(res => res.json());
+        rawProjects = projectsResp;
+        resolve();
+      })
+    ];
+
+    await Promise.all(fetches);
   }
-  worker.postMessage({ modites: rawModites, filter, date, locale });
+  rawListSource = rawModites;
+  worker.postMessage({ modites: rawListSource, filter, date, locale });
 }
 
 let minutes: number; // used by tick
 let lastFilter: string = ''; // used by onFilter
-let lastScrollOffset = 0; // used by onScroll
+let lastScrollOffset: number = 0; // used by onScroll
 
 const ModiteList: FunctionComponent<ModiteListProps & RouteComponentProps> = () => {
+  const [activeModite, setActiveModite]: [Modite, React.Dispatch<any>] = useContext(ModiteContext);
   const [modites, setModites]: [Modite[], React.Dispatch<any>] = useContext(ModitesContext);
```

Comment:
I know this change isn't in this PR but `modites` is unused. You can do something like this to skip the index in array destructuring:

```
const [ , setModites]: ....
```

Reply:
fixed